cloudconfig/podcfg/image.go

```diff
@@ -103,6 +92,16 @@ func tagImagePath(fullPath, tag string) (string, error) {
 	return imageNamed.String(), nil
 }
 
+// ImageRepoToPath returns the OCI image path.
+// If the imageRepo is empty, the default namespace is used.
+func ImageRepoToPath(imageRepo string, ver version.Number) (string, error) {
```

Comment:
this is a terrible name to be exported...  ImageRepoToPath, maybe JujudOCIImageReference.